contracts/babylon/src/contract.rs

```diff
@@ -35,10 +35,8 @@ pub fn instantiate(
     let denom = deps.querier.query_bonded_denom()?;
     let mut cfg = Config {
         network: msg.network,
-        babylon_tag: msg.babylon_tag_to_bytes()?,
         btc_confirmation_depth: msg.btc_confirmation_depth,
         checkpoint_finalization_timeout: msg.checkpoint_finalization_timeout,
```

Comment:
can we remove `network`, `btc_confirmation_depth`, and `checkpoint_finalization_timeout`?

Reply:
Not really

- `network` is used for certain address/hash handling
- k and w are used for various scenarios, e.g., checking BTC delegation expiry and fork choice